frontend/src/components/Admin/AdminShoutouts/AdminShoutouts.tsx

```diff
@@ -131,8 +137,24 @@ const AdminShoutouts: React.FC = () => {
     if (view === 'PRESENT')
       return (
         <Item.Group divided>
+          <header className={styles.presentCount}>
+            <img
+              src="https://emojis.slackmojis.com/emojis/images/1660853767/60881/meow_attention.gif?1660853767"
+              className={styles.presentCountImg}
+              alt="loading"
+            />
+            {` ${displayShoutouts.length} Shoutouts `}
+            <img
+              src="https://emojis.slackmojis.com/emojis/images/1660853767/60881/meow_attention.gif?1660853767"
```

Comment:
I don't love that these emojis are not stored on our servers. If this link were to ever break, then we'd lose the image. Can you store it in `frontend/src/static/images` please?